src/index.html

```diff
@@ -2,18 +2,50 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width,
+  <meta name="viewport" content="width=device-width,
           user-scalable=no,
           initial-scale=1.0,
           maximum-scale=1.0,
           minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Document</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:500&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
-<script type="text/javascript" src="./main.js"></script>
+    <header class="container">
+      <a href="#" class="logo">
+      <img src="images/logo.png" alt="" >
```

Comment:
correct the indentation size for `<img>`

Reply:
I can't see changes

Reply:
It`s my 3d attempt to add the correct answer. I made 2 spaces before img, and changes the menu link tag.